src/main/java/org/springframework/samples/petclinic/system/PetClinicToggles.java

```diff
@@ -13,14 +13,16 @@
 
     public static Toggle toggleFindOwnerByLastName = new Toggle(Toggle.ALWAYS_ON_WHEN_ENABLED);
     public static Toggle toggleListOfOwners = new Toggle(0.5f);
+    public static Toggle toggleFindOwnerByFirstName = new Toggle(0.5f);
 
     // TODO remove
-    public static Collection<Boolean> toggles = new ArrayList<>(Arrays.asList(toggleFindOwnerByLastName.isOn(), toggleListOfOwners.isOn()));
+    public static Collection<Boolean> toggles = new ArrayList<>(Arrays.asList(toggleFindOwnerByLastName.isOn(), toggleListOfOwners.isOn(),toggleFindOwnerByFirstName.isOn()));
```

Comment:
Remove this, and whenver A line has error, replace with

`Collection<Boolean> toggles = PetClinicToggles.getToggleValues();`